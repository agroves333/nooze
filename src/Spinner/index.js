import React from 'react';

import s from './Spinner.module.css';

const Index = () => {
  
  return (
    <div className={s.wrapper}>
      <div className={s.spinner}>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>
  );
};

export default Index;
