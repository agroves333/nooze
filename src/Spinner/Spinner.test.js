import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './index';

it('renders Spinner', () => {
  const wrapper = shallow(<Spinner />);
  expect(wrapper).toMatchSnapshot();
});
