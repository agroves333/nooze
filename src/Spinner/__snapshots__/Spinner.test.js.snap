// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Spinner 1`] = `
<div
  className="wrapper"
>
  <div
    className="spinner"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
